package/skelcd-control-openSUSE.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Jul 17 08:18:24 UTC 2018 - mlin@suse.com
+
+- Update all 15.0 related part to 15.1
```

Comment:
The changelog checker script will probably demand a bug or fate number to accept this in OBS.